projects/aws/eks-a-admin-image/provisioners/install_golang.sh

```diff
@@ -19,9 +19,19 @@ curl -L --silent $GO_URL | sudo tar xz -C /usr/local
 
 export GOROOT=/usr/local/go
 export GOPATH=/usr/local/go/packages
+export GOPROXY=direct
 
 sudo sh -c "echo 'PATH=\$PATH:$GOROOT/bin:$GOPATH/bin' >> /etc/profile.d/golang.sh"
 
 source /etc/profile.d/golang.sh
 
-go version
\ No newline at end of file
+GO=/usr/local/go/bin/go
+
+$GO version
+
+# required for e2e tests
+sudo GOPROXY=direct GOPATH=/usr/local/go/packages $GO install gotest.tools/gotestsum@latest
```

Comment:
What do you about moving this installation to a different script?
Also, why not just download the binary?

Reply:
Yeah that's fine with me. I'll try to find direct link for download and update accordingly